### PR TITLE
`GO LIVE` [Frontend docs migration] Change frontend links in top nav to the new frontend websites :rocket:

### PR DESCRIPTION
**Pull request by timonrey**
**This pull request**
changes the top nav frontend links to point to the new frontend websites. 
This will be merged along with the **go live** PR in the docs repo.
https://github.com/commercetools/commercetools-docs/pull/2472

### CODE REVIEW

**Review by emmenko — APPROVED** [review]
Thanks!

**Review by tdeekens — APPROVED** [review]
💯

File: website/src/data/top-menu.yaml [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-      href: https://docs.commercetools.com/api/graphql
+      href: /../api/graphql
     - label: Import & Export
-      href: https://docs.commercetools.com/import-export
+      href: /../import-export
     - label: SDKs & Client Libraries
-      href: https://docs.commercetools.com/sdk
+      href: /../sdk
     - label: Custom Applications
-      href: https://docs.commercetools.com/custom-applications
```
Comment (nkuehn) [inline website/src/data/top-menu.yaml:18]: @timonrey  for purposes of using the previews in the app kit docs deployment the links could have been remaining absolute here.   The other microsites are not deployed together with this one so a preview site (e.g. private beta) would have broken top nav links.

Comment (timonrey) [reply website/src/data/top-menu.yaml:18]: @nkuehn Yes, the problem was that the new feature for the top bar showing `Composable Commerce > Custom applications` was not working with absolute links, they needed to be relative paths. We can adjust the feature and change it into absolute links separately.

Comment (nkuehn) [reply website/src/data/top-menu.yaml:18]: Right.  Thank you for the clarification!  It#s an edge case, OK that way.

**Pull request merged**